openreview/venue_request/process/revisionProcess.py

```diff
@@ -72,6 +72,64 @@ def process(client, note, invitation):
                 decision_options = [s.translate(str.maketrans('', '', '"\'')).strip() for s in decision_options.split(',')]
             else:
                 decision_options = ['Accept (Oral)', 'Accept (Poster)', 'Reject']
+
+            content['send_decision_notifications'] = {
+                'description': 'Would you like to notify the authors regarding the decision? If yes, please carefully review the template below for each decision option before you click submit to send out the emails.',
+                'value-radio': [
+                    'Yes, send an email notification to the authors',
+                    'No, I will send the emails to the authors'
+                ],
+                'required': True,
+                'default': 'No, I will send the emails to the authors'
+            }
+
+            for decision in decision_options:
+                if 'Accept' in decision:
+                    content[f'{decision.lower().replace(" ", "_")}_email_content'] = {
+                        'value-regex': '[\\S\\s]{1,10000}',
+                        'description': 'Please carefully review the template below before you click submit to send out the emails. Make sure not to remove the parenthesized tokens.',
+                        'default': f'''
+Dear {{{{{{{{fullname}}}}}}}},
+
+Thank you for submitting your paper, {{submission_title}}, to {{short_name}}. We are delighted to inform you that your submission has been accepted. Congratulations!
```

Comment:
it would be nice to already replace `short_name` with `conference.short_name `here, so the PCs already see their venue's name (like we do with recruitment email templates)

Reply:
Done.